TPP.Persistence.MongoDB/Repos/BadgeRepo.cs

```diff
@@ -41,6 +41,8 @@ static BadgeRepo()
                     .SetIgnoreIfNull(true);
                 cm.MapProperty(b => b.SellingSince).SetElementName("selling_since")
                     .SetIgnoreIfNull(true);
+                cm.MapProperty(b => b.Shiny).SetElementName("shiny")
+                    .SetDefaultValue(false);
```

Comment:
If we deliberately have non-shinies not have the field in the database, this needs to also have `.SetIgnoreIfDefault` (or similar), so it doesn't become a mixed bag between "shiny: false" and "missing 'shiny' field"